src/main/java/org/thief/dto/NasaResponse.java

```diff
@@ -0,0 +1,17 @@
+package org.thief.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Setter;
+
+import java.util.Collections;
+import java.util.List;
+
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class NasaResponse {
+    private List<NasaPhoto> photos;
+
+    public List<NasaPhoto> getPhotos() {
+        return photos != null ? photos : Collections.emptyList();
```

Comment:
You can use "emptyIfNull" from apache collection utils.

Reply:
Yes, i can use it in this case.

Reply:
org.apache.commons.collections4.ListUtils#emptyIfNull

Reply:
I see, thanks, will change it.